src/index.html

```diff
@@ -1,17 +1,164 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <div class="container">
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label>
+            Surname:
+            <input
+             name="surname"
+             type="text"
+             autocomplete="off"
+            >
+          </label>
+          <label>
+            Name:
+            <input type="text"
+              name="name"
+              required
+              autocomplete="off"
+            >
+          </label>
+          <label>
+            How old are You?
+            <input type="number"
+              name="age"
+              value="12"
+              min="1"
+              max="100"
+              required
+            >
+          </label>
+          <label>
+            Full date of birth:
+            <input type="date"
+              name="birth"
+              required
+            >
+          </label>
+          <label>
+            I accept the term of agreement
+            <input type="checkbox"
+              name="agreement"
+              required
+            >
+          </label>
+        </fieldset>
+        <fieldset>
+          <legend>Registration</legend>
+          <label>
+            E-mail:
+            <input type="email"
+              name="email"
+              placeholder="email@example.com"
+              pattern="([A-z0-9_.-]{1,})@([A-z0-9_.-]{1,}).([A-z]{2,8})"
+              required
+            >
+          </label>
+          <label>
+            Password:
+            <input type="password"
+              name="password"
+              minlength="5"
+              maxlength="15"
+              required
+            >
+          </label>
+        </fieldset>
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <div class="label">
+            <span>Do you love cats? </span>
+            <input
+              id="cats_1"
+              type="radio"
+              name="cats"
+            >
+            <label for="cats_1" class="cats_1">Yes</label>
```

Comment:
Please use camelCase for attributes 
![image](https://user-images.githubusercontent.com/54936044/228381002-4e048219-8186-42d5-aa0d-e7c4bf32db2b.png)